src/Components/DraggableResizer/Documentation/DraggableResizerExampleA.md

```diff
@@ -1,6 +1,6 @@
-# 2 Panel Stateful Example
+# 3 Panel Stateful Example
 
-Here's a stateful example of `DraggableResizer` with 2 panels. Because this example contained in a stateful wrapper the example code in the JSX tab is obscured. Try looking at the State tab to see state changes in real time. 
```

Comment:
There was a mismatch in the markdown text versus storybook fixture.
2 versus 3

![Screen Shot 2022-11-08 at 1 54 57 PM](https://user-images.githubusercontent.com/10232835/200700842-3084be23-85a7-4559-9a14-965f494bf837.png)